sdk/storage/azure-storage-blob-v2/src/main/java/com/azure/v2/storage/blob/BlobClient.java

```diff
@@ -0,0 +1,482 @@
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
+     */
+    @Metadata(properties = { MetadataProperties.GENERATED })
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<Void> getAccountInfoWithResponse(Integer timeout, RequestOptions requestOptions) {
+        return this.serviceClient.getAccountInfoWithResponse(timeout, requestOptions);
+    }
+
+    /**
+     * Returns the sku name and account kind.
+     * 
+     * @param timeout The timeout parameter is expressed in seconds. For more information, see &lt;a
+     * href="https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/setting-timeouts-for-blob-service-operations"&gt;Setting
+     * Timeouts for Blob Service Operations.&lt;/a&gt;.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws StorageErrorException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     */
+    @Metadata(properties = { MetadataProperties.GENERATED })
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public void getAccountInfo(Integer timeout) {
+        this.serviceClient.getAccountInfo(timeout);
+    }
+
+    /**
+     * Returns the sku name and account kind.
+     * 
+     * @throws StorageErrorException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     */
+    @Metadata(properties = { MetadataProperties.GENERATED })
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public void getAccountInfo() {
+        this.serviceClient.getAccountInfo();
+    }
+
+    /**
+     * The Batch operation allows multiple API calls to be embedded into a single HTTP request.
+     * 
+     * @param contentLength The length of the request.
+     * @param body The body of the request.
+     * @param timeout The timeout parameter is expressed in seconds. For more information, see &lt;a
+     * href="https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/setting-timeouts-for-blob-service-operations"&gt;Setting
+     * Timeouts for Blob Service Operations.&lt;/a&gt;.
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws StorageErrorException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
+     */
+    @Metadata(properties = { MetadataProperties.GENERATED })
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<BinaryData> submitBatchWithResponse(long contentLength, BinaryData body, Integer timeout,
```

Comment:
We likely don't want to expose batch APIs directly, they're extremely complex as they're `multipart/mixed` and likely should be handled with a new Blobs Batch package. (Or some manual code in Blobs itself)